shared/common-adapters/usernames/index.tsx

```diff
@@ -128,10 +127,14 @@ function UsernameText(props: Props) {
             {/* Injecting the commas here so we never wrap and have newlines starting with a , */}
             {i !== props.users.length - 1 && (!props.inlineGrammar || props.users.length > 2) && (
```

Comment:
is inlineGrammar the same as showAnd? Can we get rid of inlineGrammar?

Reply:
no idea. who wrote this?

Reply:
Nojima